docs/04-operation-guides/troubleshooting/svls-02-failing-function-container.md

```diff
@@ -1,14 +1,13 @@
 ---
-title: Sudden container failure
-type: Troubleshooting
+title: Failing Function container
 ---
 
 The container can suddenly fail when you use the `kyma run function` command with these flags:
-    - `runtime=Nodejs12` or `runtime=Nodejs10`
+    - `runtime=Nodejs12` or `runtime=Nodejs14`
```

Comment:
in the netlify preview, all the flags are in the same line; but I think they are easier to scan as an unordered list like here.

Reply:
Thx for spotting that! I will fix the preview :)